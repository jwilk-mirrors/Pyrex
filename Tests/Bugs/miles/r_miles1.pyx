def my_raise():
	try:
		raise NotImplementedError
	except:
		raise
