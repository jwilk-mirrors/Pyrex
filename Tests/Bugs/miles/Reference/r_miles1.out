NotImplementedError
(<type 'exceptions.NotImplementedError'>, NotImplementedError(), instance of <type 'traceback'>)
